src/calibrate/requirements.txt

```diff
@@ -1,15 +1,25 @@
 Flask==1.1.2
 flask_cors==3.0.10
 flask_pymongo==2.3.0
-google-auth==1.29.0
-google-cloud-bigquery==2.13.1
-numpy==1.20.2
-pandas==1.2.4
+numpy
+pandas
 protobuf==3.15.8
 pyarrow==3.0.0
 pymongo==3.11.3
-python-dotenv==0.17.0
 scikit_learn==0.24.1
+schedule
 confluent-avro
 kafka-python
-python-dotenv
\ No newline at end of file
+python-dotenv
+gcsfs
+google-api-core
+google-auth
+google-auth-oauthlib
+google-cloud-bigquery
```

Comment:
we can specify versions for these packages

Reply:
done, thanks